src/cudadecoder/cuda-online-pipeline-dynamic-batcher.cc

```diff
@@ -167,5 +181,12 @@ void CudaOnlinePipelineDynamicBatcher::WaitForCompletion() {
   cuda_pipeline_.WaitForLatticeCallbacks();
 }
 
+int CudaOnlinePipelineDynamicBatcher::GetPendingChunks(CorrelationID corr_id) {
```

Comment:
IMO this should be named GetNumPendingChunks(), since it returns the number, not the actual chunks.

Reply:
Done